npm install fails on travis testing with node 6

`npm install` is failling with the following error during travis testing when run with node 6:

```
35.69s$ npm install 
npm ERR! Linux 4.9.6-040906-generic
npm ERR! argv "/home/travis/.nvm/versions/node/v6.12.2/bin/node" "/home/travis/.nvm/versions/node/v6.12.2/bin/npm" "install"
npm ERR! node v6.12.2
npm ERR! npm  v3.10.10
npm ERR! code EREADFILE
npm ERR! Error extracting /home/travis/.npm/clang-format/1.0.40/package.tgz archive: ENOENT: no such file or directory, open '/home/travis/.npm/clang-format/1.0.40/package.tgz'
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>
npm ERR! Please include the following file with any support request:
npm ERR!     /home/travis/build/GoogleCloudPlatform/cloud-profiler-nodejs/npm-debug.log
```

--- a/package.json
+++ b/package.json
@@ -49,7 +49,6 @@
     "@types/pretty-ms": "^3.0.0",
     "@types/request": "^2.0.7",
     "@types/sinon": "^4.0.0",
-    "clang-format": "^1.0.33",
     "codecov": "^3.0.0",
     "gts": "^0.5.0",
     "mocha": "^4.0.1",

